src/main/java/mate/academy/spring/config/WebAppInitializer.java

```diff
@@ -10,7 +10,7 @@ protected Class<?>[] getRootConfigClasses() {
 
     @Override
     protected Class<?>[] getServletConfigClasses() {
-        return new Class[]{WebConfig.class};
+        return new Class[]{WebConfig.class, SecurityConfig.class};
```

Comment:
```suggestion
        return new Class[]{WebConfig.class};
```

это должно быть в getRootConfigClasses()